components/form/AddNewItemForm.tsx

```diff
@@ -37,9 +37,13 @@ export default function AddNewItemForm({
     },
   });
 
-  const category = watch('item_type');
+  const categoryValue = watch('item_type');
```

Comment:
nice 👌

Reply:
I am not sure what do you mean by "small regression"? 

I kept the change categoryValue is consistent with the other checks and are used to see if they have been populated.